### PR TITLE
Fix authentication error when postgres is password protected

### LINKED ISSUE #282: Which is USERNAME and PASSWORD should I put no `DBBACKUP_CONNECTORS`?
I use PostgreSQL 9.5 and use md5 authentication

I set my system User ID and password on `DBBACKUP_CONNECTORS` settings.
```
DBBACKUP_CONNECTORS = {
    'default': {
        'USER': 'rightx2',
        'PASSWORD': 'MY_SYSTEM_PASSWD',
    }
}
```

Whenever I execute command, `python manage.py dbbackup`, it occurred errors:
```
dbbackup.db.exceptions.CommandConnectorError: Error running:  pg_dump super_trading --host=localhost --port=5432 --username=rightx2 --no-password --clean
b'pg_dump: [archiver (db)] connection to database "super_trading" failed: FATAL:  password authentication failed for user "rightx2"\nFATAL:  password authentication failed for user "rightx2"\n'
```

I wonder what I should put on `USER` and `PASSWORD` in  `DBBACKUP_CONNECTORS`

### PR DESCRIPTION
**Pull request by jonathan-s**
Fixes #282

### CODE REVIEW

**Comment by jonathan-s** [comment]
@NoahRR Hey there, I saw that you had issues with #282, would you mind trying out this branch and see if it works for you?

**Comment by jonathan-s** [comment]
Should add a testcase for this too -> https://dba.stackexchange.com/a/243220/132631

**Comment by ZuluPro** [comment]
👍🏾

File: dbbackup/db/postgresql.py [hunk 1]
```diff
@@ -1,5 +1,28 @@
 from dbbackup import utils
+import logging
```
Comment (ZuluPro) [inline dbbackup/db/postgresql.py:2]: Can you put `logging` before `utils` ?

File: dbbackup/tests/settings.py [hunk 2]
```diff
@@ -3,6 +3,12 @@
 """
 import os
 import tempfile
+import sys
+from dotenv import load_dotenv
```
Comment (ZuluPro) [inline dbbackup/tests/settings.py:7]: Maybe we should use django-database-url ?

Comment (jonathan-s) [reply dbbackup/tests/settings.py:7]: Using all the separate components actually works pretty well for testing in this case, as you can test any possible edge cases.

**Comment by NoahRR** [comment]
> @NoahRR Hey there, I saw that you had issues with #282, would you mind trying out this branch and see if it works for you?

@jonathan-s  Hey, yeah I'd love to. Sorry, I'm new to the programming world, how would I actually download this new branch to test if it works? I KNOW REALLY NOOB QUESTION xD

**Comment by NoahRR** [comment]
> > @NoahRR Hey there, I saw that you had issues with #282, would you mind trying out this branch and see if it works for you?
> 
> @jonathan-s Hey, yeah I'd love to. Sorry, I'm new to the programming world, how would I actually download this new branch to test if it works? I KNOW REALLY NOOB QUESTION xD

scratch that... I see the branch now xD. Let me test it

**Comment by NoahRR** [comment]
nope, I'm still a dumbass :). I cloned repo and checked out password branch, but how do I actually "install" it so I can test the new functionality. @jonathan-s I know you probably have more important things to do but I'd love to test this branch if I could >.<

**Comment by jonathan-s** [comment]
@NoahRR Nothing wrong with being a noob, we all were at some point. 

So you've cloned the repo and also changed the branch to `password`. Great! Half-way there. If you go to the folder where the cloned repo is (ie you see setup.py when you type `ls -l` in the command line). You should be able to type `pip install -e .` that would install the package in editable mode (which is excellent way of developing python packages :) ). It's pip installed and is now available everywhere you can access all the other packages that were installed the same way.

**Comment by NoahRR** [comment]
@jonathan-s I really appreciate the help (now i understand how that works)!

The new branch seems to have solved the password problem! 

... I'm still running into errors, but I believe those are due to Postgres permissions (error: permission denied for schema topology). Anyways, I don't think that is relevant and shouldn't be too hard to fix on my part. The --no-password errors are gone!

**Comment by jonathan-s** [comment]
Great to hear @NoahRR, and thanks for verifying that it works as expected.

File: dbbackup/db/postgresql.py (hunk 1 above)
Comment (jonathan-s) [reply dbbackup/db/postgresql.py:2]: Fixed.
